### ISSUE TITLE
Ruby 2.4 Stabby Lambda Syntax Error in 0.79

### ISSUE THREAD

**Issue by jherdman** [rubocop-hq/rubocop#7631]
## Expected behavior

The following is valid syntax in Ruby 2.4:

```ruby
# Some Rails model...

scope :kittens, ->(options={}) { where(breed: options[:breed]) }
```

## Actual behavior

Rubocop reports this is invalid syntax:

```
Lint/Syntax: unexpected token tLCURLY(Using Ruby 2.4 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)
```

## Steps to reproduce the problem

The above snippet should be sufficient.

## RuboCop version

```
$ [bundle exec] rubocop -V
0.79.0 (using Parser 2.7.0.1, running on ruby 2.4.1 x86_64-linux)
```

**Comment by buehmann** [comment]
This is an issue in the parser gem; see https://github.com/whitequark/parser/issues/645

**Comment by koic** [comment]
Parser gem 2.7.0.2 has been released and the issue has been resolved. Can you update parser gem to 2.7.0.2? Thanks for the feedback!
